sdk/identity/identity/test/public/node/azurePipelinesCredential.spec.ts

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AzurePipelinesCredential } from "../../../src";
+import { isLiveMode } from "@azure-tools/test-recorder";
+import { assert } from "@azure-tools/test-utils";
+import { setLogLevel } from "@azure/logger";
+setLogLevel("verbose");
+
+describe("AzurePipelinesCredential", function () {
+  const scope = "https://vault.azure.net/.default";
+  const tenantId = process.env.AZURE_SERVICE_CONNECTION_TENANT_ID!;
+
+  it.only("authenticates with a valid service connection", async function () {
+    if (!isLiveMode()) {
```

Comment:
It's not only in live mode, but outside of CI that we should skip.

People run live tests locally and this won't be available. Instead, maybe we can:
- Check if process.env.AZURE_SERVICE_CONNECTION_ID exists
- Skip if it does not exist with a message?